### DISCUSSION TITLE
Form component's reset not working select type field?

### DISCUSSION THREAD

**Discussion by ToniJ123** [sqlpage/SQLPage#671] [General]
Hi,

Form component's reset button isn't working to fields which are select type?
Could this be developed to affect also these?

At the moment we cannot clear the form in phone mode, any other way than refreshing site etc.

![form_reset_select-not-working](https://github.com/user-attachments/assets/ef9a040d-d350-4064-9b13-8e4e746f5a84)

**Comment by lovasoa** [comment]
Hi Toni! Thanks for reporting this. This is indeed a bug on our end. I opened tracking issue https://github.com/sqlpage/SQLPage/issues/672. I'll have this fixed for the next release.
